client/layout/masterbar/logged-in.jsx

```diff
@@ -82,10 +83,17 @@ class MasterbarLoggedIn extends React.Component {
 	};
 
 	renderMySites() {
-		const { domainOnlySite, hasMoreThanOneSite, siteSlug, translate } = this.props,
-			mySitesUrl = domainOnlySite
-				? domainManagementList( siteSlug )
-				: getStatsPathForTab( 'day', siteSlug );
+		const {
+				domainOnlySite,
+				hasMoreThanOneSite,
+				siteSlug,
+				translate,
+				isCustomerHomeEnabled,
+			} = this.props,
+			homeUrl = isCustomerHomeEnabled
+				? `/home/${ siteSlug }`
+				: getStatsPathForTab( 'day', siteSlug ),
```

Comment:
Shouldn't we use `config.isEnabled( 'customer-home' )` here until it's finally released?

Reply:
`isCustomerHomeEnabled` comes from `canCurrentUserUseCustomerHome` which checks `config.isEnabled( 'customer-home' )` in `isSiteEligibleForCustomerHome`. So it's buried, but it is checked.

Should we make it more explicit and check it here too?

Reply:
nah it's ok 👍